src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/FilingsService.java

```diff
@@ -141,6 +152,9 @@ private boolean setSubmissionData(Map<String, Object> data, String overseasEntit
         if (isTrustsSubmissionThroughWebEnabled) {
             data.put(TRUST_DATA, submissionDto.getTrusts());
         }
+        if (isUpdateReturn) {
```

Comment:
Not needed now and the JSON response will change in future so I would prefer to get it from the Dto and not via hash map.